v3/plugins/joaompinto/asciidoctor-vscode/2.7.5/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: joaompinto
+apiVersion: v2
+id: joaompinto.asciidoctor-vscode
+version: 2.7.5
+type: VS Code extension
+name: asciidoctor-vscode
+title: AsciiDoctor
+description: AsciiDoc Support, and code browsing https://marketplace.visualstudio.com/items?itemName=joaompinto.asciidoctor-vscode
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+spec:
+  extensions:
+  - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/joaompinto/vsextensions/asciidoctor-vscode/2.7.5/vspackage
```

Comment:
Is there any reason to have the `2.7.5` version? Once it's in the plugin registry we don't have a process for deprecating and removing it. Unless there are essential compatability/feature differences between the two, I vote we remove `2.7.5`.

Reply:
ok. Done